docs/reference/cat/nodes.asciidoc

```diff
@@ -18,7 +18,7 @@ ip        heap.percent ram.percent cpu load_1m load_5m load_15m node.role master
 --------------------------------------------------
 // TESTRESPONSE[s/3.07/(\\d+\\.\\d+( \\d+\\.\\d+ (\\d+\\.\\d+)?)?)?/]
 // TESTRESPONSE[s/65          99  42/\\d+ \\d+ \\d+/]
-// TESTRESPONSE[s/[*]/[*]/ s/mJw06l1/.+/ non_json]
+// TESTRESPONSE[s/dim/.+/ s/[*]/[*]/ s/mJw06l1/.+/ non_json]
```

Comment:
Line 101 of this file has the old `mdi` ordering.

Also, is it worth adding `l` to the docs on line 101 or is it deliberate to leave roles added by modules out of the table?

Reply:
Let's leave tweaking that to a possible follow-up, as it would include adding voting-only nodes too.